logwrap/log_wrap.py

```diff
@@ -198,6 +199,7 @@ def __init__(
         log_level: int = DEBUG,
         exc_level: int = ERROR,
         max_indent: int = 20,
+        max_iter: int = 20,
```

Comment:
IMHO, we should not change default behaviour. Better to use `0` (unlimited) as default